src/bundle/Resources/public/scss/_tag-view-select.scss

```diff
@@ -0,0 +1,63 @@
+.ibexa-tag-view-select {
+    border: calculateRem(1px) solid $ibexa-color-dark-200;
+    border-radius: $ibexa-border-radius;
+    padding: calculateRem(24px);
+    letter-spacing: calculateRem(0.12px);
```

Comment:
Are you sure `0.12px` is a proper value?

Reply:
It is correct.